mcs/class/System/System.Net.NetworkInformation/MacOsNetworkInterface.cs

```diff
@@ -36,9 +36,16 @@
 namespace System.Net.NetworkInformation {
 	internal class MacOsNetworkInterfaceAPI : UnixNetworkInterfaceAPI
 	{
-		const int AF_INET  = 2;
-		const int AF_INET6 = 30;
-		const int AF_LINK  = 18;
+		protected int AF_INET;
+		protected int AF_INET6;
```

Comment:
Please change AF_INET6 only

Reply:
...and make it `readonly`.

Reply:
@marek-safar Sure, will do.  @filipnavara If that's the right attribute, I'm happy to add it.  Thanks!